easybuild/tools/filetools.py

```diff
@@ -459,7 +459,7 @@ def download_file(filename, url, path, forced=False):
     attempt_cnt = 0
 
     # use custom HTTP header
-    url_req = urllib2.Request(url, headers={'User-Agent': 'EasyBuild'})
+    url_req = urllib2.Request(url, headers={'User-Agent': 'EasyBuild',  "Accept" : "text/html"})
```

Comment:
@oliviermattelaer What does the `text/html` part mean here exactly?

Reply:
I think we should use `*/*` rather than `text/html`, to accept all types... Cfr. https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html

Please make that change, then it should be OK to merge this.

Reply:
Sure. Done.

Thanks,

Olivier